nconnect.go

```diff
@@ -91,31 +114,15 @@ func Run(opts *config.Opts) {
 		persistConf.Identifier = config.RandomIdentifier()
 		opts.Identifier = persistConf.Identifier
 		shouldSave = true
+	} else {
+		persistConf.Identifier = opts.Identifier
```

Comment:
Why modify persistConf when it should not be saved?

Reply:
Restored.